2022-06-03 16:58:53,900 INFO    MainThread:160877 [wandb_setup.py:_flush():75] Loading settings from /home/gabriel/.config/wandb/settings
2022-06-03 16:58:53,900 INFO    MainThread:160877 [wandb_setup.py:_flush():75] Loading settings from /home/gabriel/mestrado/disciplinas/machine-learning/mlops/mlops/wandb/settings
2022-06-03 16:58:53,900 INFO    MainThread:160877 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-06-03 16:58:53,900 WARNING MainThread:160877 [wandb_setup.py:_flush():75] Could not save program above cwd: /home/gabriel/anaconda3/envs/mlops/bin/uvicorn
2022-06-03 16:58:53,901 INFO    MainThread:160877 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': None, 'program': '/home/gabriel/anaconda3/envs/mlops/bin/uvicorn'}
2022-06-03 16:58:53,901 INFO    MainThread:160877 [wandb_init.py:_log_setup():438] Logging user logs to /home/gabriel/mestrado/disciplinas/machine-learning/mlops/mlops/wandb/run-20220603_165853-1hgsnj67/logs/debug.log
2022-06-03 16:58:53,901 INFO    MainThread:160877 [wandb_init.py:_log_setup():439] Logging internal logs to /home/gabriel/mestrado/disciplinas/machine-learning/mlops/mlops/wandb/run-20220603_165853-1hgsnj67/logs/debug-internal.log
2022-06-03 16:58:53,902 INFO    MainThread:160877 [wandb_init.py:init():472] calling init triggers
2022-06-03 16:58:53,902 INFO    MainThread:160877 [wandb_init.py:init():475] wandb.init called with sweep_config: {}
config: {}
2022-06-03 16:58:53,902 INFO    MainThread:160877 [wandb_init.py:init():525] starting backend
2022-06-03 16:58:53,903 INFO    MainThread:160877 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-06-03 16:58:53,907 INFO    MainThread:160877 [backend.py:ensure_launched():219] starting backend process...
2022-06-03 16:58:53,914 INFO    MainThread:160877 [backend.py:ensure_launched():224] started backend process with pid: 160905
2022-06-03 16:58:53,916 INFO    MainThread:160877 [wandb_init.py:init():534] backend started and connected
2022-06-03 16:58:53,919 INFO    MainThread:160877 [wandb_init.py:init():598] updated telemetry
2022-06-03 16:58:54,016 INFO    MainThread:160877 [wandb_init.py:init():629] communicating run to backend with 30 second timeout
2022-06-03 16:58:57,231 INFO    MainThread:160877 [wandb_run.py:_on_init():1922] communicating current version
2022-06-03 16:58:57,868 INFO    MainThread:160877 [wandb_run.py:_on_init():1926] got version response upgrade_message: "wandb version 0.12.17 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-06-03 16:58:57,868 INFO    MainThread:160877 [wandb_init.py:init():660] starting run threads in backend
2022-06-03 16:59:02,914 INFO    MainThread:160877 [wandb_run.py:_console_start():1896] atexit reg
2022-06-03 16:59:02,915 INFO    MainThread:160877 [wandb_run.py:_redirect():1769] redirect: SettingsConsole.REDIRECT
2022-06-03 16:59:02,917 INFO    MainThread:160877 [wandb_run.py:_redirect():1774] Redirecting console.
2022-06-03 16:59:02,923 INFO    MainThread:160877 [wandb_run.py:_redirect():1830] Redirects installed.
2022-06-03 16:59:02,923 INFO    MainThread:160877 [wandb_init.py:init():685] run started, returning control to user process
2022-06-03 17:03:01,320 INFO    MainThread:160877 [wandb_run.py:_atexit_cleanup():1865] got exitcode: 1
2022-06-03 17:03:01,322 INFO    MainThread:160877 [wandb_run.py:_restore():1837] restore
2022-06-03 17:03:08,816 INFO    MainThread:160877 [wandb_run.py:_on_finish():1995] got exit ret: None
2022-06-03 17:03:08,917 INFO    MainThread:160877 [wandb_run.py:_restore():1837] restore
